package/scripts/precook_test.py

```diff
@@ -0,0 +1,237 @@
+import os
+import time
+import random
+import gc
+import shutil
+
+pid = 1729
+
+
+def make_stat(pid, proc_pid, fixed_value, rand=False):
+    stat_fname = os.path.join(proc_pid, "stat")
+    if rand:
+        with open(stat_fname, "w") as f:
+            # Writing some pid and process name although never used
+            f.write(str(pid) + " ")  # 0 pid
+            f.write("(python3)" + " ")  # 1 Executable filename
+            # 2 to 12 are never used (as are 0 and 1) -> set to 0
+            for i in range(11):
+                f.write(str(0) + " ")
+
+            # Define upper limit for random generation of integers for utime and stime
+            time_lim = 10000000
+            # 13 utime - Time in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 14 stime - Time in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 15 cutime - Time of waited-for children in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 16 cstime - Time of waited-for children in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+
+            # 17 and 18 are never used -> set to 0
+            f.write("0 0 ")
+
+            # 19 num_threads - Number of threads of the process
+            num_threads = 1
+            f.write(str(num_threads) + " ")
+
+            # 20 never used -> set to 0
+            f.write("0 ")
+
+            # 21 starttime (uptime in utils.h) (last field that is read by any monitor)
+            f.write(str(random.randint(1, time_lim)))
+    else:
+        with open(stat_fname, "w") as f:
+            # Writing some pid and process name although never used
+            f.write(str(pid) + " ")  # 0 pid
+            f.write("(python3)" + " ")  # 1 Executable filename
+            # 2 to 12 are never used (as are 0 and 1) -> set to 0
+            for i in range(11):
+                f.write(str(0) + " ")
+
+            # 13 utime - Time in user mode
+            f.write(str(fixed_value) + " ")
+            # 14 stime - Time in kernel mode
+            f.write(str(fixed_value) + " ")
+            # 15 cutime - Time of waited-for children in user mode
+            f.write(str(fixed_value) + " ")
+            # 16 cstime - Time of waited-for children in kernel mode
+            f.write(str(fixed_value) + " ")
```

Comment:
```py
f"{fixed_value} " * 4 
```

One for @amete too - I think in this script we can use f-strings as it is not run by librarians or end users of `prmon`